File: Service_eHospital_DongNai_A/Implement/ChiPhiKCBNoiTru_BHYTService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class ChiPhiKCBNoiTru_BHYTService : BaseService<ChiPhiKCBNoiTru_BHYT, IChiPhiKCBNoiTru_BHYTRepository>
    , IChiPhiKCBNoiTru_BHYTService
    {
    private readonly IChiPhiKCBNoiTru_BHYTRepository _ChiPhiKCBNoiTru_BHYTRepository;
    public ChiPhiKCBNoiTru_BHYTService(IChiPhiKCBNoiTru_BHYTRepository ChiPhiKCBNoiTru_BHYTRepository) : base(ChiPhiKCBNoiTru_BHYTRepository)
    {
    _ChiPhiKCBNoiTru_BHYTRepository = ChiPhiKCBNoiTru_BHYTRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/ChiPhiKCBNoiTru_BHYT.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ChiPhiKCBNoiTru_BHYT } from './ChiPhiKCBNoiTru_BHYT.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ChiPhiKCBNoiTru_BHYTService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Id', 'KhamBenh_Id', 'BenhAn_Id', 'BenhNhan_Id', 'DoiTuong_Id', 'SoBHYT', 'NgayVaoVien', 'NgayRaVien', 'SoNgay', 'TienPhong', 'Thuoc', 'DichTruyen', 'XetNghiem', 'SieuAm', 'DienTim', 'DienNao', 'XQuang', 'CT', 'NoiSoi', 'ThuThuat', 'VTYT', 'DichVuYTe', 'Khac'];

    List: ChiPhiKCBNoiTru_BHYT[] | undefined;
    ListFilter: ChiPhiKCBNoiTru_BHYT[] | undefined;
    FormData!: ChiPhiKCBNoiTru_BHYT;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ChiPhiKCBNoiTru_BHYT";
    }
}

